src/services/caching/cache.interfaces.ts

```diff
@@ -6,9 +6,18 @@ export interface IWizdomPageViewCache {
     ExecuteCached<T>(key: string, func: Function, expiresInMilliseconds: number) : T;
 }
 
+export interface IWizdomTimestamps {
+    get(key?: string) : Promise<number>;
+    addMappings(key: string, ...mappings: string[]) : void;
+}
+export interface IWizdomTimestampsResolver {
+    resolve(timestamps: {[key:string]:number}) : void;
+}
```

Comment:
I think we should keep all public methods in PascalCase to be consistent with the exposed methods on the other interfaces.